src/methods.js

```diff
@@ -16,6 +16,7 @@ import { set } from 'lodash';
 
 export default {
   addMultiSigAddress: { version: '>=2.2.0' },
+  addNode: { version: '>=2.2.0' },
```

Comment:
Would you mind further alphabetizing this so that all of them are alphabetical?

Reply:
If you'd like to merge it now I'd be more than happy to go through the list afterwards and alphabetise it completely.  :)